tests/testthat/test_ExtractData_Soils.R

```diff
@@ -18,23 +19,26 @@ do_skip <- c(
   # mimmics skip_on_appveyor():
   identical(tolower(Sys.getenv("APPVEYOR")), "true"))
 
-suppressWarnings(is_online <-
-                   !inherits(try(close(url(getOption("repos"), open = "r")), silent = TRUE),
-                             "try-error"))
 
-
-if (!any(do_skip) && is_online) {
+if (!any(do_skip)) {
   # =============================================================================
   # Tests designed to test the underlining structures created
   # from soil extraction functions.
   # =============================================================================
   
-  # set stage manually so that future changes won't cause the test to fail ======
   # setup file paths
   fnames_in <- environment()
   fnames_in$fslayers <- file.path("/home/natemccauslin/Desktop/Dryland Ecology/rSFSW2/tests/test_data/TestPrj4/1_Input/SWRuns_InputData_SoilLayers_v9.csv")
   fnames_in$fsoils <- "/home/natemccauslin/Desktop/Dryland Ecology/rSFSW2/tests/test_data/TestPrj4/1_Input/datafiles/SWRuns_InputData_soils_v12.csv"
+  # directory where extertnal soils data is located
   dir_ex_soil <- "/media/natemccauslin/SOILWAT_DATA/GIS/Data/Soils/"
+  
+  # if any of the file paths above are invalid, skip all tests and setup
+  if (!all(file.exists(c(fnames_in$fslayers, fnames_in$fsoils, dir_ex_soil)))) {
```

Comment:
You don't check that `dir_ex_soil` contains the required content for function `extract_soil_ISRIC250m`. Let's assume that `dir_ex_soil` exists but the person has not downloaded the specific dataset, then this still fails because of 
```
dir.ex.gridded <- file.path(dir_ex_soil, "ISRIC", "GriddedGlobalV5")
  # stop program execution if folder path is incorrect
  if (!dir.exists(dir.ex.gridded)) stop(paste0("Folder '", dir.ex.gridded,
"' does not exist"))
```

Reply:
- The unit test shouldn't be testing if the directory contains the files before the function is called because that is done in `extract_soil_ISRIC250m` so it is done regardless of how the function is being called. Basically if I were to do that I'd be checking it twice. 

- Having the function fail because the person doesn't have the correct dataset is by design.

Reply:
Having the function fail because a user doesn't have the files is correct; but having the unit tests fail because a developer doesn't have the files is bad -- the unit tests should skip.